### ISSUE TITLE
Exception in Lombok plugin (0.28-2019.3)

### ISSUE THREAD

**Issue by n4v4j0** [mplushnikov/lombok-intellij-plugin#713]
### Version information

 * **IDEA Version:** IntelliJ IDEA 2019.3 (Ultimate Edition)
Build #IU-193.5233.102, built on November 27, 2019
Licensed to *****
Subscription is active until *****
Runtime version: 11.0.4+10-b520.11 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 1957M
Cores: 8
Registry: 
Non-Bundled Plugins: Create Jira Ticket, JFormDesigner, Jira Browser, Key Promoter X, Lombook Plugin, PlantUML integration, com.kukido.eclipser, com.przemyslawj.mockito.generator, org.mapstruct.intellij, org.pmesmeur.sketchit, org.zalando.intellij.swagger

 * **JDK Version:** java version "1.8.0_211"
Java(TM) SE Runtime Environment (build 1.8.0_211-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.211-b12, mixed mode)

 * **OS Type & Version:** Windows 10 Pro, Version: 1803, OS build: 18134.1009
 * **Lombok Plugin Version:** 0.28-2019.3
 * **Lombok Dependency Version:** Gradle: org.projectlombok.lombok:lombok-1.18.8.jar

### Stacktrace

```

com.intellij.diagnostic.PluginException: class de.plushnikov.intellij.plugin.psi.LombokLightParameter; valid=true; method.valid=true [Plugin: Lombook Plugin]
	at com.intellij.ide.plugins.PluginManagerCore.createPluginException(PluginManagerCore.java:399)
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:12)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:58)
	at com.intellij.codeInsight.daemon.impl.quickfix.ChangeMethodSignatureFromUsageFix.getNewParametersInfo(ChangeMethodSignatureFromUsageFix.java:328)
	at com.intellij.codeInsight.daemon.impl.quickfix.ChangeMethodSignatureFromUsageFix.isAvailable(ChangeMethodSignatureFromUsageFix.java:150)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.addAvailableFixesForGroups(ShowIntentionsPass.java:149)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.lambda$getAvailableFixes$0(ShowIntentionsPass.java:82)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getAvailableFixes(ShowIntentionsPass.java:82)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getActionsToShow(ShowIntentionsPass.java:291)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.doCollectInformation(ShowIntentionsPass.java:220)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:55)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:429)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:997)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:421)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:397)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:157)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:395)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
Caused by: com.intellij.psi.PsiInvalidElementAccessException: Element: class com.intellij.psi.impl.source.PsiJavaCodeReferenceElementImpl #JAVA  because: different providers: com.intellij.psi.DummyHolderViewProvider{myVirtualFile=LightVirtualFile: \DummyHolder, content=VirtualFileContent{size=0}}(39284922); com.intellij.psi.SingleRootFileViewProvider{myVirtualFile=LightVirtualFile: \DummyHolder, content=VirtualFileContent{size=0}}(6e41ffa)
invalidated at: see attachment; Type element reference of class com.intellij.psi.impl.source.PsiFieldImpl #PsiFieldImpl #Language: JAVA; augmenters=[com.intellij.psi.impl.source.JShellPsiAugmentProvider@6d27a6b4, de.plushnikov.intellij.plugin.provider.LombokAugmentProvider@5c7a37b0, com.intellij.lang.aspectj.augment.AjPsiAugmentProvider@52dba0cd]
	at com.intellij.psi.impl.source.PsiClassReferenceType.resolveGenerics(PsiClassReferenceType.java:177)
	at com.intellij.psi.impl.source.PsiClassReferenceType.resolve(PsiClassReferenceType.java:124)
	at com.intellij.psi.util.PsiUtil.ensureValidType(PsiUtil.java:1268)
	at com.intellij.psi.util.PsiUtil.ensureValidType(PsiUtil.java:1257)
	at com.intellij.codeInsight.daemon.impl.quickfix.ChangeMethodSignatureFromUsageFix.getNewParametersInfo(ChangeMethodSignatureFromUsageFix.java:322)
	... 25 more

```

**Comment by NCNecros** [comment]
I confirm this bug on:
IntelliJ IDEA 2019.3 (Community Edition)
Build #IC-193.5233.102, built on November 28, 2019
Runtime version: 11.0.4+10-b520.11 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 982M
Cores: 4
Registry: 


Plugin version: 0.15.17.2
Lib version: lombok-1.18.10.jar

**Comment by mplushnikov** [comment]
@NCNecros your plugin version is very old. Please update to a new one : 0.28

**Comment by mplushnikov** [comment]
@n4v4j0 Your exception is duplicate of #649. Do you have some reproducible example for it? It would help a lot to find out some solution!

**Comment by n4v4j0** [comment]
This Exception seems to be not critical. I have not expirienced any limitation of usability of lombok-plugin. I noticed this Exception in my IntelliJ-Report-Window, and dont know its first occurence. So I can not give you example how to reproduce it. 
@NCNecros : my plugin-version IS the latest available : 0.28-2019.3 
Lombok-Version is also new: 1.18.8
